### ISSUE TITLE
Next batch of mailing lists/group emails for scrf.io

### ISSUE THREAD

**Issue by ghettodev** [smartcontractresearchforum/research_team#217]
Please get me a list of the existing mailing lists you are using and a wish list for new ones. Keep in mind that there should be a naming convention here and that team leads will be expected to maintain access to them. 

- should be tied to group of activities that support internal operations
- avoid creating lists for short term projects

**Comment by marvino3** [comment]
Some important aliases may include:
1) hiring@ - job opportunities, collecting resumes, etc. 
2) discovery@ - eventual leads, campaigns, PR, etc (aka general marketing or sales initiatives) 
3) socials@ - could be helpful to have a place for people to send in post ideas and responses
4) support@ - responding to any questions, issues, and thoughts from the community (ex: bugs, where to find something)
5) I would change invoice@ to a more general finance email like billing@ but I get if that is a pain
Open to thoughts here. Thank you.

**Comment by ghettodev** [comment]
Do each of the leads need a vertical@ list?

**Comment by eleventh19** [comment]
I agree with @marvino3's suggestions. One q - does it make sense to have both discovery and socials to start? Or phase one in as we get more people doing socials (unless that's just an easier process than the current one) 

In terms of invoice -> billing, I agree with you overall but unless we see it as invoice@ being out of line in terms of discovery, it will be pain so it'd be great to stick with invoice@ 

@ghettodev to your point, I think that partnerships@ can be an overall outreach one
why do we need it: to capture anyone who wants to reach out to us and to explore potential partnership

**Comment by zubepaul** [comment]
I have been brainstorming potential uses of an engagement@ and can not think of an engagement type of inbound or outbound emails. Perhaps an onboarding@, so I will check in with Tamar and Valerie regarding potential uses they see there. 

Re: point above with partnerships, I think that generally makes sense. Maybe having it focus as collaboration@ widens its perceived use a bit?

**Comment by marvino3** [comment]
@eleventh19 I'm happy to start with discovery@ initially and then adding socials if we need to down the line

And yeah, I figured re: invoice vs billing. It will be more of a pain to switch later if we decide. Something like billing@ or finance@ seems more comprehensive and representative. But ultimately up to you.
